src/commands/Danke.js

```diff
@@ -1,17 +1,30 @@
+/** @typedef {import('discord.js/src/structures/Message')} Message */
+
 import BaseCommand from './BaseCommand.js';
 
 export default class Danke extends BaseCommand {
+    /**
+     *
+     */
     constructor() {
         super({
-            aliases: ['danke', 'd']
-        })
+            aliases: ['danke', 'd'],
```

Comment:
Pass `aliases` as separate param. See BaseCommand constructor params in `BaseCommand.js` line 8 .